<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShareButtonBar</class>
 <widget class="QWidget" name="ShareButtonBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#content_widget{
background:#1d2129;
border-radius:30px;
}

QPushButton{
background:transparent;
border-radius: 8px;
font-family: 'Inter';
font-style: normal;
font-weight: 400;
font-size: 12px;
line-height: 22px;
text-align: center;
color: #C9CDD4;
border:none;
}

QPushButton:hover{
background: rgba(255, 255, 255, 0.05);
}

QPushButton:pressed{
background: rgba(255, 255, 255, 0.1);
}

QPushButton:disabled{
 background: rgba(255, 255, 255, 0.3);
}



#line{
background:#4E5969;
}



</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="content_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>533</width>
       <height>48</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_camera">
        <property name="minimumSize">
         <size>
          <width>56</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>56</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#btn_camera{
text-align:left;
padding-left:8px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/video_call_resource.qrc">
          <normaloff>:/img/videocall_camera_on</normaloff>:/img/videocall_camera_on</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_mic">
        <property name="minimumSize">
         <size>
          <width>56</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>56</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#btn_mic{
text-align:left;
padding-left:8px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/video_call_resource.qrc">
          <normaloff>:/img/videocall_mic_on</normaloff>:/img/videocall_mic_on</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_data">
        <property name="minimumSize">
         <size>
          <width>56</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>56</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/video_call_resource.qrc">
          <normaloff>:/img/videocall_real_time_data</normaloff>:/img/videocall_real_time_data</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_share">
        <property name="minimumSize">
         <size>
          <width>56</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>56</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#btn_share{
text-align:left;
padding-left:8px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/video_call_resource.qrc">
          <normaloff>:/img/videocall_share</normaloff>:/img/videocall_share</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_beauty">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>56</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>56</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/video_call_resource.qrc">
          <normaloff>:/img/videocall_beauty</normaloff>:/img/videocall_beauty</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_setting">
        <property name="minimumSize">
         <size>
          <width>56</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>56</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/video_call_resource.qrc">
          <normaloff>:/img/videocall_setting</normaloff>:/img/videocall_setting</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_end">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>72</width>
          <height>28</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#btn_end{
background: #F53F3F;
border-radius: 14px;
color: #FFFFFF;
}

#btn_end:hover{
background: #f57676;
}
#btn_end:pressed{
background:#ad2f2f
}

</string>
        </property>
        <property name="text">
         <string>结束共享</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/video_call_resource.qrc"/>
 </resources>
 <connections/>
</ui>
